### ISSUE TITLE
k_fold_cross_validation Throwing an Error

### ISSUE THREAD

**Issue by invictus2010** [CamDavidsonPilon/lifelines#1157]
Hi there!

I was following along the example [here](https://lifelines.readthedocs.io/en/latest/Examples.html#model-selection-using-lifelines) for `Model Selection.`

The following code is featured:
`print(np.mean(k_fold_cross_validation(cph, df, duration_col='T', event_col='E', scoring_method="concordance_index")))`

*Issue #1: The use of `duration_col='T'` appears contrary to the documentation on the `k_fold_cross_validation` method found [here](https://lifelines.readthedocs.io/en/latest/lifelines.utils.html#lifelines.utils.k_fold_cross_validation). I believe this is correctly written as `print(np.mean(k_fold_cross_validation(cph, df, 'T', event_col='E', scoring_method="concordance_index")))`

*Issue #2: Whether or not I include `duration_col=` in `k_fold_cross_validation`, I receive the following error:

```
/usr/local/lib/python3.6/dist-packages/lifelines/utils/__init__.py in k_fold_cross_validation(fitters, df, duration_col, event_col, k, scoring_method, fitter_kwargs)
    773         for fitter, scores in zip(fitters, fitter_scores):
    774             # fit the fitter to the training data
--> 775             fitter.fit(training_data, duration_col=duration_col, event_col=event_col, **fitter_kwargs)
    776             scores.append(fitter.score(testing_data, scoring_method=scoring_method))
    777 

/usr/local/lib/python3.6/dist-packages/lifelines/utils/__init__.py in f(model, *args, **kwargs)
     52         def f(model, *args, **kwargs):
     53             cls.set_censoring_type(model, cls.RIGHT)
---> 54             return function(model, *args, **kwargs)
     55 
     56         return f
TypeError: fit() got an unexpected keyword argument 'duration_col'
```
I think this suggests that duration_col is mistakenly hard-coded somewhere in the package. I'll take a look, but I'm a relative newbie. Maybe I'm just doing something wrong?

Here's the code that I run 

```
print(np.mean(k_fold_cross_validation(kmf, users, duration_col='T', event_col='E', scoring_method="concordance_index")))
print(np.mean(k_fold_cross_validation(cure_model, users, duration_col='T', event_col='E', scoring_method="concordance_index")))
```

**Comment by invictus2010** [comment]
I intend to close this issue since it was brought up [here ](https://github.com/CamDavidsonPilon/lifelines/issues/571#issuecomment-446691227)in Issue #571 . 

TLDR; this isn't the intended use of univariate fitters.
